File: 151001/Danko/discussion/src/main/java/com/example/discussion/dto/MessageRequestTo.java
package com.example.discussion.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
public class MessageRequestTo {
    private Long id;
    private Long storyId;
    @NotBlank
    @Size(min = 2, max = 32)
    private String content;
    private String method;
    private String country;
}

File: 151004/Korol/Lab4/publisher/DC_Lab1/DC_Lab1/DTO/MessageRequestTo.cs
using DC_Lab1.DTO.Interface;
using System.Text.Json.Serialization;

namespace DC_Lab1.DTO
{
    public class MessageRequestTo : IRequestTo
    {
        public int Id { get; set; }

        public int? StoryId { get; set; }

        public string? Content { get; set; }
    }
}
